metrics/disttask.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"strconv"
+
+	"github.com/pingcap/tidb/disttask/framework/proto"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	lblTaskStatus  = "status"
+	lblTaskType    = "task_type"
+	lblTaskID      = "task_id"
+	lblSubTaskID   = "subtask_id"
+	lblSchedulerID = "scheduler_id"
+)
+
+var (
+	// DistDDLSubTaskCntGauge is the gauge of dist ddl subtask count.
+	DistDDLSubTaskCntGauge *prometheus.GaugeVec
+	// DistDDLSubTaskStartTimeGauge is the gauge of dist ddl subtask start time.
+	DistDDLSubTaskStartTimeGauge *prometheus.GaugeVec
+)
+
+// InitDistDDLMetrics initializes disttask metrics.
+func InitDistDDLMetrics() {
```

Comment:
What about `import into` metrics?

Reply:
This is the distributed framework metrics, which contain import into if import into use distributed framework.

Import into metrics is [here](https://github.com/pingcap/tidb/pull/46749)

Reply:
How about renaming it to `InitDistTaskMetrics()`?

Reply:
It is ok to me